src/environment/stella_environment.cpp

```diff
@@ -139,12 +142,12 @@ reward_t StellaEnvironment::act(Action player_a_action, Action player_b_action)
   //  past the terminal state
   for (size_t i = 0; i < m_frame_skip; i++) {
     
-    // Stochastically drop actions, according to repeatProb
-    // @todo -- don't use rand()
-    if (rand() / double(RAND_MAX + 1.0) >= repeat_prob)
+    // Stochastically drop actions, according to m_stochasticity
+    if (m_rand_gen.next() / double(MAX_RAND + 1.0) >= m_stochasticity){
```

Comment:
Instead of using MAX_RAND, doesn't it make more sense to implement a method Random::nextDouble() which hides away this mess (& guards against bugs)? This is e.g. how Java and the boost Mersenne twister operate.